danger/Dangerfile

```diff
@@ -0,0 +1,27 @@
+# Check if branch name follows format
+type_pattern = /(fix|feat|refactor|improve|build|ci|docs|chore|test|tests|revert)/
+ticket_pattern = /(MAG|SDKCF)-\d{3,5}/ # Main single ticket in capital letters that can connect this branch to a board tracker ID
```

Comment:
I don't think this should be included in common config because we are limiting the pattern to our ticket format.
And this config is opensource and should not be limited.

Reply:
Understood. Should we keep Dangerfile in each repo then?

Reply:
Hmm
I think we can retain it here. But we need to make the `ticket_pattern` as gradle property.

Reply:
Updated to inject ticket pattern as variable `DANGER_TICKET_NUM_REGEX`